nautobot/ipam/models.py

```diff
@@ -867,10 +1039,6 @@ def clean(self):
 
         if self.address:
 
-            # /0 masks are not acceptable
-            if self.address.prefixlen == 0:
-                raise ValidationError({"address": "Cannot create IP address with /0 mask."})
```

Comment:
IIRC there's logic in `ipam/factory.py` that's preventing it from creating /0 prefixes, might be able to remove that now?

Reply:
Definitely not looking forward to revising the factory. 😩